spec/struct.dd

```diff
@@ -1373,6 +1373,229 @@ $(H3 $(LNAME2 implicit-copy-constructors, Implicit Copy Constructors))
 
     $(P If the generated copy constructor fails to type check, it will receive the `@disable` attribute.)
 
+$(H2 $(LEGACY_LNAME2 StructMoveConstructor, struct-move-constructor, Struct Move Constructors))
+
+    $(P Move constructors are very much like copy constructors. The difference is that copy
+    constructors make a copy of the original, while move constructors move the contents of
+    the original, and the lifetime of the original ends.)
+
+    $(NOTE Do not use postblits in the same struct with move constructors.)
```

Comment:
If we have to have a warning like this, we should probably not support it.

Reply:
I plan on investigating if postblits should be allowed if there are move constructors.

Reply:
What I'm wondering here is perhaps we have been thinking about postblit a bit wrong.

Instead of it being the third hook, make it the common denominator of both copy/move constructors. So it is always called after they are.